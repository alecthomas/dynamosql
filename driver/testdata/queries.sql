SELECT * FROM gamescores WHERE UserId = "101"
SELECT * FROM gamescores WHERE UserId = "101" AND TopScore > 30
SELECT * FROM gamescores WHERE UserId = "103" AND GameTitle BETWEEN "Galaxy" AND "Meteor" AND TopScore > 1000
