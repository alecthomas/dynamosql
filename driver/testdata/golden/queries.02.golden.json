{
  "Query": "SELECT * FROM gamescores WHERE UserId = \"103\" AND GameTitle BETWEEN \"Galaxy\" AND \"Meteor\" AND TopScore > 1000",
  "Results": [
    "GameTitle,Losses,TopScore,UserId,Wins",
    "Galaxy Invaders,3,2317,103,40"
  ]
}
